crowdsec-docs/docs/profiles/format.md

```diff
@@ -105,6 +104,17 @@ It is a list of `models.Decision` objects. The following fields, when present, a
  - `type` : defines the type of the remediation that will be applied by available bouncers, for example `ban`, `captcha`
  - `value` : define a hardcoded value for the decision (ie. `1.2.3.4`)
 
+### `duration_expr`
+
+```yaml
+duration_expr: "Sprintf('%dh', (GetDecisionsCount(Alert.GetValue()) + 1) * 4)"
+```
+
+If the profile applies, and the `duration_expr` is valid and generate a valid [golang's duration](https://pkg.go.dev/time#ParseDuration) it will replace the decision duration.
```

Comment:
If the profile applies, and the `duration_expr` generates a valid [golang's duration](https://pkg.go.dev/time#ParseDuration) it will replace the decision duration.

It can be used to have custom duration. For example, you can have an increased duration every time an attacker comes back.
It relies on [expr helpers](/expr/helpers.md).